farm/eval.py

```diff
@@ -7,14 +7,23 @@
 from sklearn.metrics import classification_report
 from torch.utils.data import DataLoader
 
-from farm.metrics import compute_metrics
+from farm.metrics import compute_metrics, _correct_bio_encodings
 from farm.utils import to_numpy
 from farm.utils import MLFlowLogger as MlLogger
 from farm.modeling.adaptive_model import AdaptiveModel
 
 logger = logging.getLogger(__name__)
 
 
+def _token_classification_report(labels, preds, digits):
```

Comment:
Please test if this report also still works with LM finetuning. There we use it to report performance on the masked language model task. If it doesn't work we might need two separate report types here.